airflow/models/connection.py

```diff
@@ -380,3 +381,17 @@ def extra_dejson(self) -> Dict:
                 self.log.error("Failed parsing the json for conn_id %s", self.conn_id)
 
         return obj
+
+
+def get_connections(conn_id: str) -> List[Connection]:
```

Comment:
```suggestion
def _get_secret_connections(conn_id: str) -> List[Connection]:
```

Reply:
True .. They are only used internally and in tests :)

Reply:
Well this one is actually used in Base Hook. but I can incorporate it there.

Reply:
Well. It seems even more appropriate to make them static methods in Connection/Variable respectively